src/BlazorUI/Bit.BlazorUI.Extras/Components/FullCalendar/BitFullCalendar.scss

```
@@ -0,0 +1,2592 @@
+/* BitFullCalendar */
+
+@import "../../../Bit.BlazorUI/Styles/functions.scss";
```

Stylelint (17.13.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)